platform/kube/client.go

```diff
@@ -360,7 +361,7 @@ func modelToKube(km model.KindMap, obj *model.Config) (*Config, error) {
 	return out, nil
 }
 
-func protoToMap(msg proto.Message) (map[string]interface{}, error) {
+func protoToMap(msg proxycf.ProxyConfig) (map[string]interface{}, error) {
```

Comment:
don't specialize this method - it's meant to be used for all proto messages (there is a descriptor field in each generated Go source)